flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
If you get Stockfish to build from source, it will be possible to make it available for aarch64 tooo.